koji_containerbuild/cli.py

```diff
@@ -103,7 +103,27 @@ def handle_container_build(options, session, args):
         parser.error(_("Exactly two arguments (a build target and a SCM URL "
                        "or archive file) are required"))
         assert False
-    clikoji.activate_session(session)
+
+    # Koji API has changed - activate_session requires two arguments
+    # _running_in_bg has been moved to koji_cli.lib
+    try:
```

Comment:
Had to roll this back - `clikoji` is not yet set by that time, so it doesn't have `_running_in_bg` option